src/lib/datetime.ts

```diff
@@ -36,13 +36,40 @@ function ensureFullICU() {
 	}
 }
 
+export function formatCourseDurationForCourseInlcFaceToFace(d: number) {
```

Comment:
d = duration?

Reply:
yup, I can't change it as the whole page was built using d as duration